src/status_im/subs/root.cljs

```diff
@@ -195,3 +195,5 @@
 
 ;;keycard
 (reg-root-key-sub :keycard :keycard)
+
+(reg-root-key-sub :log-level/current-pre-login-log-level :log-level/current-pre-login-log-level)
```

Comment:
Hi, please can you elaborate why `current-pre-login...` not just `pre-login-log...`?